Sources/SpelledPitch/PitchSpeller/Wetherfield/Wetherfield.swift

```diff
@@ -68,23 +74,38 @@ struct PitchSpeller {
         self.pitches = pitches
         self.parsimonyPivot = parsimonyPivot
         self.pitchNodes = PitchSpeller.internalNodes(pitches: pitches)
-        self.flowNetwork = FlowNetwork(source: -2, sink: -1, internalNodes: pitchNodes)
+        self.flowNetwork = FlowNetwork(
+            source: Cross<Int, Tendency>(-1, .down),
+            sink: Cross<Int, Tendency>(-1, .up),
+            internalNodes: pitchNodes
+        )
     }
 
     /// - Returns: An array of `SpelledPitch` values in the order in which the original
     /// unspelled `Pitch` values are given.
     func spell() -> [SpelledPitch<EDO12>] {
 
         var assignedNodes: [AssignedNode] {
-            let (sourceSide, sinkSide) = flowNetwork.minimumCut
+            var (sourceSide, sinkSide) = flowNetwork.minimumCut
+            sourceSide.remove(flowNetwork.source)
+            sinkSide.remove(flowNetwork.sink)
             let downNodes = sourceSide.map { index in AssignedNode(index, .down) }
             let upNodes = sinkSide.map { index in AssignedNode(index, .up) }
             return downNodes + upNodes
         }
 
         return assignedNodes
-            .sorted { $0.index < $1.index }
-            .dropFirst(2)
+            .sorted {
+                switch (
+                    $0.index.a < $1.index.a,
+                    $0.index.a == $1.index.a,
+                    UInt8($0.index.b.rawValue) < UInt8($1.index.b.rawValue)
```

Comment:
My feeling is that we shouldn't have to convert this to a `UInt8` and extract out the `rawValue`. This comparison seems like it may be better suited as part of the type (`Tendency`, unless I'm mistake) itself.

Reply:
💯This was a case of saying "whatever you say" to the compiler ! I am hoping to deprecate this with #126